app/jobs/queue_trainee_updates_job.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class QueueTraineeUpdatesJob < ApplicationJob
+  queue_as :dttp
+
+  INVALID_STATES = %w[
+    draft
+    recommended_for_qts
+    withdrawn
+    deferred
+    qts_awarded
+  ].freeze
```

Comment:
https://stackoverflow.com/questions/37799296/ruby-what-does-the-comment-frozen-string-literal-true-do

Reply:
This is an array though not a string so it still needs freezing.